### ISSUE TITLE
NVDA is not announcing the upper and lower limit of spin button.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#14952]
### Steps to reproduce:

1. Turn on screen reader NVDA.
2. Hit enter on to [https://speech.microsoft.com/audiocontentcreation](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fspeech.microsoft.com%2faudiocontentcreation&data=05%7c01%7cv-roopesht%40microsoft.com%7c3e9c323caa934e49629408db40ba7b41%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c638174941901379029%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=1S%2fZ%2B1QfK%2BADwIECYDjM1NcZ74o6gGnFmp4SyLdhuNA%3D&reserved=0).
3. Tab to the "Try the lite version of Audio Content Creation" section 
4. Tab to the "Break" section.
5. Tab to move to 'Custom' tab in Break.
6. Tab to the spin button present.
7. Check if the limit upper/lower limit of the spin button is being announced by screen reader.

### Actual behavior:
When tabbing to the custom editable spin button in break section, screen reader is not announcing the upper/lower limit of the spin button which should be announced when entering to the spin button.

### Expected behavior:
When tabbing to the custom editable spin button in break section, screen reader should be announcing the upper/lower limit of the spin button.

#### Does the issue still occur after restarting your computer? yes

[35974_NVDA.webm](https://github.com/nvaccess/nvda/assets/93735775/43abbbc8-02a1-4d6f-ad88-9df07767b722)

**Comment by seanbudd** [comment]
Could you provide an example speech string how you think it should be announced?

**Comment by msftedad** [comment]
Hi Any update on this issue??
